packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -409,6 +433,8 @@ class WebReleaseBundle extends Target {
       depfile,
       environment.buildDir.childFile('flutter_assets.d'),
     );
+    final File assetManifestFile = outputDirectory.childFile('AssetManifest.bin');
+    await assetManifestFile.copy(environment.buildDir.childFile('AssetManifest.bin').path);
```

Comment:
Why do this? Arent you writing the asset manifest into the entrypoint?

Reply:
Removed this in commit [remove outdated addition to outputs of WebReleaseBundle](https://github.com/flutter/flutter/pull/130419/commits/1fae6be4c46f79074dedab4cc261db9682f39512)